tests/e2e-chainsaw/v1beta2/teardown/redis-sentinel/sentinel.yaml

```diff
@@ -8,8 +8,9 @@ spec:
   podSecurityContext:
     runAsUser: 1000
     fsGroup: 1000
-  redisSentinelConfig:
-    redisReplicationName : redis-replication
+   # If we specify the redisReplicationName, we should wait for the redis cluster to be ready before deploying the sentinel
```

Comment:
Why don't we use the chinsaw wait operation in another step so that we can wait till the redis replication is ready @drivebyer

Reply:
> Why don't we use the chainsaw wait operation in another step so that we can wait till the redis replication is ready @drivebyer

In my opinion, in this test case, it is sufficient to install Sentinel independently, and then test the StatefulSet and Service. We do not specify redisReplicationName, the Sentinel StatefulSet and Service will still be created as well.

Reply:
We have already test RedisReplication with RedisSentinel in https://github.com/OT-CONTAINER-KIT/redis-operator/tree/master/tests/e2e-chainsaw/v1beta2/ha-setup :) @shubham-cmyk